src/tests/specs/inkjs/compiler/Core.spec.ts

```diff
@@ -24,10 +23,10 @@ describe("Core parsers", () => {
     expect(parser.index).toBe(24);
   });
 
-  it("parses newLine", () => {
-    const parser = new InkParser(`moo text and 
-then an -> happens
-and what ?`);
+  it.only("parses newLine", () => {
+    const parser = new InkParser(
+      `moo text and \nthen an -> happens\nand what ?`
```

Comment:
I turned those into `\n`-bound strings because my VS Code removes trailing spaces on save. Prettier is a lot smarter and can detect that trailing spaces should be kept in a template string, but to prevent future headaches I made the whitespace more explicit.

Reply:
Works for me. Templates with new lines were mainly a side effect of me copy-pasting the examples around, this is indeed cleaner that way !

Reply:
Happy to hear that!